content/docs/rfcs/23/README.md

```diff
@@ -355,11 +355,6 @@ The implementation SHOULD follow http://rfc.zeromq.org/spec:29/PUBSUB for the se
 
 When using ZMTP, message filtering SHALL happen at the publisher side (the PUB or XPUB socket). To create a subscription, the SUB or XSUB peer SHALL send a SUBSCRIBE message, which has this grammar:
 
-message = *message-more message-last
```

Comment:
why is this block being removed?

Reply:
It has nothing to do with subscription per se, right?

I don't know why that block was copied from 

https://github.com/zeromq/rfc/blob/f8160affb154edea301d2fd1a912d4289e921c1a/content/docs/rfcs/23/README.md#L124

without `<code>` formatting

Reply:
It has nothing to do with subscription per se, right?

I don't know why the block was copied from

https://github.com/zeromq/rfc/blob/f8160affb154edea301d2fd1a912d4289e921c1a/content/docs/rfcs/23/README.md#L124

without `<code>` formatting

Reply:
Ah, I see what you mean - looked at the diff only, not the rendered markdown